app/src/main/java/com/example/theperiodpurse/AppScreen.kt

```diff
@@ -44,6 +50,13 @@ fun ScreenApp(
             }
         }
     ) { innerPadding ->
+        Image(
+            painter = painterResource(id = R.drawable.background),
+            contentDescription = null,
+            modifier = Modifier.fillMaxWidth().fillMaxHeight().fillMaxSize(),
```

Comment:
nitpick: I think `fillMaxSize()` already does `fillMaxWidth` and `fillMaxHeight`, so having the other two would be redundant.